app/src/main/java/com/rentminder/SaveButton.kt

```diff
@@ -1,31 +1,17 @@
 package com.rentminder
 
 import android.widget.Toast
-import androidx.compose.animation.animateContentSize
-import androidx.compose.animation.core.LinearOutSlowInEasing
```

Comment:
Removed all imports that were not being used. These cluttered up the imports if you need to review them later. Additionally, they bloated the line numbers.